.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Something is crashing or not working as intended
+
+---
+
+*Please consider making a Pull Request if you are capable of doing so.*
+
+**Library Version:**
+
+2.x.x
+  
+**Describe the Bug:**
+
+A clear description of what is the bug is.
+
+**To Reproduce:**
```

Comment:
I think that "Steps to reproduce the bug" is more descriptive

Reply:
Done!